cli/tools/jupyter/install.rs

```diff
@@ -1,40 +1,31 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
 
-use deno_core::anyhow::bail;
-use deno_core::anyhow::Context;
 use deno_core::error::AnyError;
 use deno_core::serde_json;
 use deno_core::serde_json::json;
 use std::env::current_exe;
-use std::io::ErrorKind;
 use std::io::Write;
 use std::path::Path;
-use tempfile::TempDir;
+
+use runtimelib::dirs::user_data_dir;
 
 const DENO_ICON_32: &[u8] = include_bytes!("./resources/deno-logo-32x32.png");
 const DENO_ICON_64: &[u8] = include_bytes!("./resources/deno-logo-64x64.png");
 const DENO_ICON_SVG: &[u8] = include_bytes!("./resources/deno-logo-svg.svg");
 
 pub fn status() -> Result<(), AnyError> {
-  let output = std::process::Command::new("jupyter")
-    .args(["kernelspec", "list", "--json"])
-    .output()
-    .context("Failed to get list of installed kernelspecs")?;
-  let json_output: serde_json::Value =
-    serde_json::from_slice(&output.stdout)
-      .context("Failed to parse JSON from kernelspec list")?;
+  let user_data_dir = user_data_dir()?;
 
-  if let Some(specs) = json_output.get("kernelspecs") {
-    if let Some(specs_obj) = specs.as_object() {
-      if specs_obj.contains_key("deno") {
-        log::info!("✅ Deno kernel already installed");
-        return Ok(());
-      }
-    }
-  }
+  let kernel_spec_dir_path = user_data_dir.join("kernels").join("deno");
+  let kernel_spec_path = kernel_spec_dir_path.join("kernel.json");
```

Comment:
I wonder if all of this could be abstracted into a high-level API in `runtimelib` like so:
```
install_kernel("deno", "v1.43.4");
install_kernel("deno", "latest");
```
This would allow installing multiple versions of a kernel with ease.